<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>课表转换byWell404</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ScheduleTrans">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>转换课表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ScheduleCheck">
       <property name="text">
        <string>检索源文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,0,1">
     <item>
      <widget class="QListWidget" name="FileList"/>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,2">
       <item>
        <widget class="QPlainTextEdit" name="BasicInfo">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string>本程序由Well404提供
联系方式QQ：1070330078
版本信息：0.1.0alpha
————————————
操作方法：
1.请将课表原文件置于&quot;Input&quot;文件夹中，并点击右上角的检索源文件
2.在左侧的表格中核对文件名，点击转换课表。
3.在 &quot;output&quot;文件夹中可获取对应的导入文件。
————————————
注意事项：
1.本程序需要在安装有Excel的电脑中运行，金山WPS可能会产生未知的错误。
2.如遇任何BUG或问题，请联系原作者，并提供课表源文件，以便解决问题并完善程序。
3.本程序适用于新教务系统导出的课表，旧课表请参考旧版转换器。</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="ErrorInfo">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
